### PR TITLE
Help for Proxy issues in using http tool

### PR DESCRIPTION
**Pull request by greeshmab**
For proxy issues even after downloading http and configuring it, we are supposed to use proxy everytime we use http command.

### CODE REVIEW

File: docs/tutorials/first-steps.rst [hunk 1]
```diff
@@ -29,6 +29,7 @@ We'll start with a relatively simple data model:
 
 Using the `httpie <http://httpie.org>` tool we can post a sample record in the
 ``tasks`` collection:
+For proxy issues we can use  "http --proxy=http:http://usrname:passwrd@host:port COMMAND" instead of "http COMMAND"
```
Comment (Natim) [inline docs/tutorials/first-steps.rst]: I am sure there should be a way to define the PROXY in the ENV. See https://github.com/jkbrzt/httpie#proxies

Comment (Natim) [inline docs/tutorials/first-steps.rst]: Please consider adding a note stating that:

Please [consider reading httpie documentation](https://github.com/jkbrzt/httpie#proxies) for more information If you need to configure a proxy for instance.

Comment (greeshmab) [inline docs/tutorials/first-steps.rst]: Yes, This would be more helpful

Comment (Natim) [inline docs/tutorials/first-steps.rst]: Please update your ``greeshmab:patch-1`` branch to make this little change so that we can merge it :+1:

**Comment by leplatrem** [comment]
Thanks for this contribution :)

**Pull request merged**